cmd/otel-allocator/internal/server/server.go

```diff
@@ -84,8 +87,13 @@ func (s *Server) setRouter(router *gin.Engine) {
 	router.UnescapePathValues = false
 	router.Use(s.PrometheusMiddleware)
 
+	router.GET("/", s.IndexHandler)
+	router.GET("/collector", s.CollectorHTMLHandler)
+	router.GET("/job", s.JobHTMLHandler)
+	router.GET("/target", s.TargetHTMLHandler)
+	router.GET("/targets", s.TargetsHTMLHandler)
```

Comment:
I think we should maybe put these HTML handlers behind a path like `/debug` to distinguish it from the API routes.